appveyor.yml

```diff
@@ -2,9 +2,10 @@ version: '{build}'
 branches:
   only:
   - master
+  - debug-appveyor
 clone_folder: c:\projects\llassetgen
 image:
-- Visual Studio 2013
+#- Visual Studio 2013
```

Comment:
Why can't we use VS 2013 anymore? And if we can't, I think this should be documented somewhere, maybe the Readme

Reply:
i couldnt figure out, why only the vs 2013 debug build fails, but since we have 2017 it should be ok to just drop vs 2013. I dont think this should be documented somewhere, also because it wasnt documented before that we are trying to build for vs 2013.